core/runtime.rs

```diff
@@ -180,6 +183,10 @@ pub struct RuntimeOptions {
   /// executed tries to load modules.
   pub module_loader: Option<Rc<dyn ModuleLoader>>,
 
+  /// JsRuntime modules, not to be confused with ES modules
+  /// these are deno modules containing JS + ops
+  pub modules: Vec<Box<dyn JsRuntimeModule>>,
```

Comment:
These names should change as well - `JsRuntimeModule` cannot contain ES modules as source files, only plain old scripts, because currently snapshotting doesn't support ES modules.

Reply:
Yes, that's pretty clear, it's linked to the above naming conversation, I'll change the names once there's some consensus.

Reply:
is there a v8 bug ticket for ES modules not being snapshottable? it would be nice if support was added. i searched v8 & chromium bug trackers but didn't turn up anything.

Reply:
@rektide, not, that is not a V8 bug, it's lacking Deno implementation.